src/guides/v2.4/install-gde/prereq/mysql.md

```diff
@@ -33,327 +33,93 @@ If your web server and database server are on different hosts, perform the tasks
-
-The following procedure is based on [Install MySQL Server 5.6 in CentOS 6.x and Red Hat 6.x Linux](http://sharadchhetri.com/2013/12/26/install-mysql-server-5-6-in-centos-6-x-and-red-hat-6-x-linux/){:target="_blank"}.
-
-1. *CentOS 6* Install the MySQL database:
-
-   ```bash
-   yum -y update
-   ```
-
-   ```bash
-   sudo wget http://repo.mysql.com/mysql-community-release-el6-5.noarch.rpm && sudo rpm -ivh mysql-community-release-el6-5.noarch.rpm
-   ```
-
-   ```bash
-   sudo yum -y install mysql-server
-   ```
-
-1. *CentOS 7* Install the MySQL database:
-
-   ```bash
-   yum -y update
-   ```
-
-   ```bash
-   sudo wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm && sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm
-   ```
-
-   ```bash
-   sudo yum -y install mysql-server
-   ```
-
-1. Start MySQL:
-
-   ```bash
-   service mysqld start
-   ```
-
-1. Set a password for the <tt>root</tt> user and set other security-related options. Enter the following command and follow the prompts on your screen to complete the configuration:
-
-   ```bash
-   mysql_secure_installation
-   ```
-
-1. Verify the MySQL server version:
-
-   ```bash
-   mysql -u root -p
-   ```
-
-   Messages similar to the following display:
-
-   ```terminal
-   Welcome to the MySQL monitor.  Commands end with ; or \g.
-   Your MySQL connection id is 15 Server version: 5.6.23 MySQL Community Server (GPL)
-
-   Copyright (c) 2000, 2015, Oracle and/or its affiliates. All rights reserved.
-
-   Oracle is a registered trademark of Oracle Corporation and/or its affiliates. Other names may be trademarks of their respective owners.
-
-   Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
-   ```
+Magento disabled the regular validation behavior by setting SET SQL_MODE='' in `/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php:424.`. With validation disabled, it is possible that MySQL will truncate data. In MySQL, the Query behavior has changed: `Select * on my_table where IP='127.0.0.1'` will no longer return any results because the IP address is now properly seen as a string, rather than an integer.
 
-1. If you expect to import large numbers of products into Magento, you can configure MySQL to use the [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} parameter. We recommend a value of at least 16MB.
+## Upgrading from MySQL 5.7 to MySQL 8
 
-   {% include install/mysql_max-allowed-packet-centos.md %}
+In order to properly update your MySQL from version 5.7 to version 8, the process must be done in the following order:
 
-1. Configure the Magento database instance as discussed in the next section.
+1. Upgrade Magento to 2.4.0.
+   Test everything makes sure your system work as expected.
+1. Enable maintenance mode.
+1. Make a backup from the database.
+1. Update MySQL to version 8.
+1. Import the dump into MySQL.
```

Comment:
Is the "dump" the same thing as the "backup" mentioned in step 3? If so, use the same terminology.